README.md

```diff
@@ -371,6 +371,17 @@ Params:
 - `message`: the message to show. May use template expressions (e.g.
   `{{.my_input}}`).
 
+  The print action has special access to an extra template variable named
+  `flags` containing the values of some of the command line flags. This can be
+  useful to print a message like
+  `Template rendering is done, now please to go the {{.flags.dest}} directory and run a certain command.`
```

Comment:
This feels a little over engineered to me, given the current stage of this project. We can probably save this for later when there are things beyond `dest` we'd like to print.

Reply:
Not against wrapping flags to pass them around though.

Reply:
I have a use case for this for bootstrapping a new GCP org from the setup template.

Reply:
I can imagine it but I was thinking a more basic version where we just the relative path to point to a readme :)

Reply:
I think it's good as-is. It's important to show helpful messages to the user.